osgar/drivers/lora.py

```diff
@@ -1,8 +1,16 @@
 """
   LoRa - Long Range Radio
 
-  Current protocol uses numbers for DeviceID (1, 2, ... 5) for source identification and '|' for data separation.
-  The whole packet is terminated by '\n'
+  This module handles communication with LoRa devices and automatic re-transmission
+  of received messages so that all robots work as re-transmission stations.
+  The messages have limited size 40 bytes.
+
+  At the moment driver supports only transmission of "pose2d" messages so the control
+  center (and all neighbor robots) know about each other.
+
+  Current protocol uses numbers for DeviceID (1, 2, ... 5) for source identification
+  and '|' for data separation. The whole packet is terminated by '\n'. The DeviceID
+  is added in front. The message history is collected in reverse order.
```

Comment:
It is not obvious from anything that the message history is collected inside the message. Is that even true? If not, where is it collected? Can you add an example message?

Reply:
OK, example added